src/modules/TodoList.js

```diff
@@ -0,0 +1,38 @@
+import Todo from './Todo.js';
+
+export default class TodoList {
+  constructor(todolist) {
+    this.todolist = todolist;
+  }
+
+  addTask(description, index) {
+    const newTask = new Todo(description, index);
+    this.todolist.push(newTask);
+  }
+
+  removeTask(index) {
+    this.todolist.splice((index - 1), 1);
+    const updatedTodoList = this.todolist.map((object) => {
+      if (object.index > index) {
+        const ind = object.index - 1;
+        return { ...object, index: ind };
+      }
+      return object;
+    });
+    this.todolist = updatedTodoList;
+  }
+
+  editTask(description, index) {
+    const updatedTodoList = this.todolist.map((object) => {
+      if (parseInt(object.index, 10) === parseInt(index, 10)) {
+        return { ...object, description };
+      }
+      return object;
+    });
+    this.todolist = updatedTodoList;
+  }
+
+  getTaskByIndex(index) {
+    return this.todolist.filter((obj) => parseInt(obj.index, 10) === parseInt(index, 10))[0];
+  }
```

Comment:
Hi @sajibarafatsiddiqui , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media2.giphy.com/media/YKdm91XONLTaw/giphy-downsized-medium.gif"/>

## Highlights
✔️ Descriptive PR
✔️ Set up app with webpack 📌
✔️ Good commit history
✔️ No linter errors
✔️ JavaScript bet practices
✔️ Html & CSS best practices

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
You did a great job creating those functions! 💯 However, I have a couple of suggestions in order to improve it:

    [OPTIONAL]:  It would be great if you Refactor your methods and functions to arrow functions in the TodoList.js file! 🤓 🛠️
ONAL]:  It would be great if you Refactor your methods and functions to arrow functions in the TodoList.js file! 🤓 🛠️